src/libopensc/card-iasecc.c

```diff
@@ -644,8 +644,6 @@ iasecc_init_cpx(struct sc_card *card)
 	_sc_card_add_rsa_alg(card, 1024, flags, 0);
 	_sc_card_add_rsa_alg(card, 2048, flags, 0);
 
-	LOG_TEST_RET(ctx, sc_enum_apps(card), "Enumerate apps failed");
-
```

Comment:
I get a "permission denied" when opening the chromium.org URL.

I do confirm that it works with this patch. I am confused because some other drivers (for example edo_init()) does the same calls.

OK to remove this line, but please, if you can provide some backgrounds about "why is it called by other drivers ?", it would be welcomed.

Reply:
Yes. The bugs are restricted for the developers but they will open 30 days after releasing fix.